modules/tinymce/src/core/main/ts/Deprecations.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Arr, Obj } from '@ephox/katamari';
+
+import { EditorSettings, RawEditorSettings } from './api/SettingsTypes';
+import Tools from './api/util/Tools';
+
+const deprecatedSettings = (
+  'autoresize_on_init,content_editable_state,convert_fonts_to_spans,inline_styles,padd_empty_with_br,block_elements,' +
+  'boolean_attributes,editor_deselector,editor_selector,elements,file_browser_callback_types,filepicker_validator_handler,' +
+  'force_hex_style_colors,force_p_newlines,gecko_spellcheck,images_dataimg_filter,mode,move_caret_before_on_enter_elements,' +
+  'non_empty_elements,self_closing_elements,short_ended_elements,special,spellchecker_select_languages,spellchecker_whitelist,' +
+  'tab_focus,table_responsive_width,text_block_elements,text_inline_elements,toolbar_drawer,types,validate,whitespace_elements'
+).split(',');
+
+const deprecatedPlugins = 'bbcode,colorpicker,contextmenu,fullpage,legacyoutput,spellchecker,textcolor'.split(',');
```

Comment:
Any particular reason to not just have these as arrays?

Reply:
They end up being larger minified and I was trying to keep this as minimal as possible.